src/wristpy/core/orchestrator.py

```diff
@@ -159,14 +99,132 @@ def run(
         Literal["ggir", "gradient"],
     ] = "gradient",
     epoch_length: Union[int, None] = 5,
-    verbosity: int = 30,
-) -> Results:
+    verbosity: int = logging.WARNING,
```

Comment:
I don't see a None return. If that's correct then no need for an Optional return.